libraries/pbrlib/genglsl/lib/mx_microfacet_sheen.glsl

```diff
@@ -27,9 +27,13 @@ float mx_imageworks_sheen_brdf(float NdotL, float NdotV, float NdotH, float roug
 // Rational quadratic fit to Monte Carlo data for Imageworks sheen directional albedo.
 float mx_imageworks_sheen_dir_albedo_analytic(float NdotV, float roughness)
 {
-    float a = 5.25248 - 7.66024 * NdotV + 14.26377 * roughness;
-    float b = 1.0 + 30.66449 * NdotV + 32.53420 * roughness;
-    return a / b;
+    vec2 r = vec2(13.67300, 1.0) +
```

Comment:
@niklasharrysson. Pinging to check this is the correct version here.

Reply:
Yep, this is the new improved one.